frontend/cli_to_rails_proxy.js

```diff
@@ -1,9 +1,10 @@
 const PROXY_HOSTNAME = process.env.PROXY_HOSTNAME || 'localhost';
+const PORT = process.env.PORT || '3000';
 
 const PROXY_CONFIG = [
   {
     "context": ['/**'],
-    "target": `http://${PROXY_HOSTNAME}:3000`,
+    "target": `http://${PROXY_HOSTNAME}:${PORT}`,
```

Comment:
I'm wondering what this `PROXY_HOSTNAME` is supposed to be.

In `frontend/cli_to_rails_proxy.js`, it's used with port 3000 so it seems to be the backend host, and in `frontend/package.json` it's used with port 4200 so it looks like it's the frontend host.

Shouldn't this `PROXY_HOSTNAME` be removed entirely and use `HOST` or `FE_HOST` instead?
If so, the documentation at `docs/development/running-tests/README.md:704` should be updated too.

`PROXY_HOSTNAME` is also used in `docker-compose.yml`, but I do not fully understand how it is supposed to work in combination with `HOST` and `PORT` as I'm not using this docker setup regularly.

Reply:
I was really not sure about touching `PROXY_HOSTNAME`, as it is in compose file, but you may be right and it should be replaced with `HOST`.
I'm also not sure if there is a need for separate `HOST` and `FE_HOST` variables, maybe there should be just `HOST`, `PORT` and `FE_PORT`.

Reply:
Probably someone from the Titan team can help, as they are all using docker.

Reply:
@cbliard @toy In the context of the docker setup `PROXY_HOSTNAME` refers to the container host name, for instance `backend`, for the backend container. So `localhost` wouldn't work here. So this should remain and not be replaced with `HOST`.

Reply:
Thanks for the explanation. That makes more sense. 
So when `PROXY_HOSTNAME` is not set, it should default to either `HOST` or `FE_HOST` then?

Reply:
The fallback should be `HOST`.